module-10/10.2/client/package.json

```diff
@@ -38,7 +36,10 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^6.4.8",
+    "@testing-library/react": "^16.0.0",
     "http-proxy-middleware": "^3.0.0",
+    "jest": "^27.5.1",
```

Comment:
See notes above: react-scripts already includes jest.

Unsure if there are other reasons we'd need jest as a top-level dependency